### PR TITLE
Valid example fix

### PR DESCRIPTION
**Pull request by acunniffe**
## 🍗 Description
_What does this PR do? Anything folks should know?_

## 📚 References
_Links to relevant docs (Notion, Twist, GH issues, etc.), if applicable._

## 👹 QA
_How can other humans verify that this PR is correct?_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,5 +29,5 @@
       "yarn prettier --write"
     ]
   },
-  "stableVersion": "0.39.2"
+  "stableVersion": "0.39.3"
```
Comment (niclim) [inline package.json:32]: The stable version is incorrect here

**Comment by acunniffe** [comment]
Customer said this works!

File: projects/standard-rulesets/src/examples/requireValidExamples.ts [hunk 2]
```diff
@@ -93,11 +94,17 @@ function strictAdditionalProperties(schema: any, skip: boolean = false) {
...
-
-      Object.values(schema).forEach((s) => {
-        strictAdditionalProperties(s, false);
-      });
+    } else if (
+      schema.hasOwnProperty('type') &&
+      schema.type === 'object' &&
+      !schema.hasOwnProperty('unevaluatedProperties')
+    ) {
+      // schema.unevaluatedProperties = false;
```
Comment (Alexandre-Herve) [inline projects/standard-rulesets/src/examples/requireValidExamples.ts:102]: Is this left commented out on purpose?

File: projects/optic/src/commands/dereference/dereference.ts [hunk 3]
```diff
@@ -52,18 +56,45 @@ const getDereferencedSpec = async (
...
       const specJson = parsedFile.jsonLike;
 
+      if (includeExtensions.length) {
+        Object.entries(specJson.paths).forEach(([path, operations]) => {
+          Object.entries(operations!).forEach(([key, operation]) => {
+            if (key === 'parameters') return;
+            if (
+              operation &&
+              !includeExtensions.some((extension) =>
+                Boolean(operation[extension])
```
Comment (Alexandre-Herve) [inline projects/optic/src/commands/dereference/dereference.ts:81]: ⛏️ Boolean is superfluous here

**Review by Alexandre-Herve — APPROVED** [review]
Syntax looks fine, I don't understand what's happening though

**Pull request merged**